### PR TITLE
Bug/sc 111415 Fix for empty timestamps in transpose & sync recipe

### PR DESCRIPTION
**Pull request by alexbourret**

### CODE REVIEW

File: python-connectors/pi-system_event-frames-search/connector.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 from osisoft_constants import OSIsoftConstants
 from safe_logger import SafeLogger
 from osisoft_plugin_common import PISystemConnectorError, RecordsLimit, get_credentials, build_requests_params, assert_time_format, get_advanced_parameters
+# from osisoft_pagination import Pagination
```
Comment (ehysi) [inline python-connectors/pi-system_event-frames-search/connector.py:8]: should this be removed or uncommented ?

**Review by ehysi — COMMENTED** [review]
LGTM . Tests are OK . 

Just some remarks on the commented code as I am not sure if this was intentional or not to be included in this PR. If it is indeed desired , we should remove those lines.

File: python-connectors/pi-system_event-frames-search/connector.py [hunk 2]
```diff
@@ -63,8 +65,10 @@ def generate_rows(self, dataset_schema=None, dataset_partitioning=None,
                 **self.config
             )
             headers = self.client.get_requests_headers()
+            # pagination = Pagination(self.client.get, self.client.get, self.database_endpoint + "/eventframes", headers=headers, params=params, error_source="generate_rows")
```
Comment (ehysi) [inline python-connectors/pi-system_event-frames-search/connector.py:68]: same as above

File: python-connectors/pi-system_event-frames-search/connector.py [hunk 3]
```diff
@@ -63,8 +65,10 @@ def generate_rows(self, dataset_schema=None, dataset_partitioning=None,
...
             next_page_url = self.database_endpoint + "/eventframes"
             is_first = True
+            # while pagination.has_content():
```
Comment (ehysi) [inline python-connectors/pi-system_event-frames-search/connector.py:71]: same as above

File: python-connectors/pi-system_event-frames-search/connector.py (hunk 1 above)
Comment (alexbourret) [reply python-connectors/pi-system_event-frames-search/connector.py:8]: Indeed I'm looking at changing how the pagination is done throughout the whole plugin but it's not completed yet. I'll remove it for now...

Comment (alexbourret) [reply python-connectors/pi-system_event-frames-search/connector.py]: Removed in [0e8d31f](https://github.com/dataiku/dss-plugin-osisoft/pull/17/commits/0e8d31feacfaa01e6ac699ff34e6a6d07754c94f)

File: python-connectors/pi-system_event-frames-search/connector.py (hunk 2 above)
Comment (alexbourret) [reply python-connectors/pi-system_event-frames-search/connector.py]: Removed in [0e8d31f](https://github.com/dataiku/dss-plugin-osisoft/pull/17/commits/0e8d31feacfaa01e6ac699ff34e6a6d07754c94f)

File: python-connectors/pi-system_event-frames-search/connector.py (hunk 3 above)
Comment (alexbourret) [reply python-connectors/pi-system_event-frames-search/connector.py]: Removed in [0e8d31f](https://github.com/dataiku/dss-plugin-osisoft/pull/17/commits/0e8d31feacfaa01e6ac699ff34e6a6d07754c94f)

**Review by ehysi — APPROVED** [review]

**Pull request merged**